content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Quickstart"
+description: "SeaChat Intro and getting started"
+lead: ""
+date: 2024-03-08 10:43:51.069 +0100
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+---
+* In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+
+In this tutorial, we will create a SeaChat agent from scratch. This AI Agent will help us answer frequently asked questions about parking lots. We will walk you through the process of creating an AI agent, adding knowledge to its knowledge base, and testing and fine-tuning the agent to ensure it's ready for deployment.
+
+## Step 1: Create a Workspace 
+If this is your first time logging into SeaChat, you'll need to create a workspace to start the process. In this tutorial, we'll create a new workspace called **My Work Space**. Enter the name of your workspace and let's dive in!
```

Comment:
"If this is your first time logging into SeaChat, you'll need to create a workspace to start the process. I named my new workspace **My Work Space**. You can give your workspace a different name!"

Add a pic of workspace creation (with the workspace name "My Work Space" filled in